CHANGELOG.rst

```diff
@@ -2,6 +2,16 @@
 Change history
 ==============
 
+2.2.2 (2024-10-01)
+------------------
+
+**Bugfixes and QOL**
+
+* fix API schema not showing caused by CSP errors
```

Comment:
It would be nice to have issue\PR numbers for changes, it will help in the future to find why some change was made (like today with LOG_REQUEST in OZ)

Reply:
Added